File: foundation/foundation 2 month/lesson8/homework/ex_1.c
#include <stdio.h>
void check(int a)
{
	if(a==0)
	return ;
	check(a-1);
	printf("%d ",a);
}

int main()
{
	int n;
	printf("N ni kiriting: ");
	scanf("%d",&n);
	check(n);
}

File: foundation/foundation 3 month/lesson7/homework/ex_1.py
def check(ms):
	j=0
	while j<len(ms):
		if  ms[j]=='':
			ms.pop(j)
		else:
			j+=1
	return ms
k=int(input(" Nechta element Kiriting: "))
arr=list()
for x in range(1,k+1):
	name=input(f"{x} chi elemeni ni kiriting: ")
	arr.append(name)
ans=check(arr)
print(ans)

